pkg/domain/domain.go

```diff
@@ -1482,24 +1482,8 @@ func (do *Domain) Init(
 		return err
 	}
 
-	startReloadTime := time.Now()
 	// step 3: domain reload the infoSchema.
-	err = do.Reload()
-	if err != nil {
-		return err
-	}
-
-	sub := time.Since(startReloadTime)
-	// The reload(in step 2) operation takes more than ddlLease and a new reload operation was not performed,
-	// the next query will respond by ErrInfoSchemaExpired error. So we do a new reload to update schemaValidator.latestSchemaExpire.
-	if sub > (do.schemaLease / 2) {
-		logutil.BgLogger().Warn("loading schema and starting ddl take a long time, we do a new reload", zap.Duration("take time", sub))
-		err = do.Reload()
-		if err != nil {
-			return err
-		}
-	}
-	return nil
+	return do.Reload()
```

Comment:
What if the reload failed with err? here I saw that still need domain.Reload().

Reply:
Previously, when the reload exceeded half of the lease, we needed to reload again, but now we don’t need to. 
If the reload fails here, we will retry at the upper layer.

Reply:
Previously, when the reload exceeded half of the lease, we needed to reload again, but now we don’t need to. 
If the reload fails here, we will retry at the upper layer.